cms/static/sass/views/_group-configuration.scss

```diff
@@ -273,23 +275,13 @@
 
           // add a group is below with groups styling
           .action-primary {
-            @include blue-button;
-            @include transition(all .15s);
-            @extend %t-action2;
-            @extend %t-strong;
-            display: inline-block;
-            padding: ($baseline/5) $baseline;
-            text-transform: uppercase;
+            @extend %btn-primary-blue;
+            padding: ($baseline*.3) $baseline;
```

Comment:
Oh man, what's 20/3?! :P Any chance we could make this a cleaner end value after the division (``$baseline/4``)? If not, I'd ditch the reliance on the $baseline variable and go with the actual pixel value instead.